tinygrad/helpers.py

```diff
@@ -56,7 +56,7 @@ def __ge__(self, x): return self.value >= x
   def __gt__(self, x): return self.value > x
   def __lt__(self, x): return self.value < x
 
-DEBUG, IMAGE = ContextVar("DEBUG", 0), ContextVar("IMAGE", 0)
+DEBUG, NOSTAT, IMAGE = ContextVar("DEBUG", 0), ContextVar("NOSTAT", 0), ContextVar("IMAGE", 0)
```

Comment:
Why do I ever want NOSTAT?

Reply:
Removed it. With the `sym_infer` cache results do not differ so much.